File: Projeto_6/src/usuario.py
class Usuario:
    def __init__(self, nome: str, login: str, tipo_usuario):
        self._nome = nome
        self._login = login
        self._tipo_usuario = tipo_usuario

    @property
    def nome(self):
        return self._nome

    @property
    def tipo_usuario(self):
        return self._tipo_usuario

File: Projeto_6/src/Usuario.cpp
#include "Usuario.h"

Usuario::Usuario(const std::string& nome, const std::string& email, TipoUsuario tipo)
    : nome(nome), email(email), tipo(tipo) {}

Usuario::~Usuario() {}

std::string Usuario::getNome() const { return nome; }
std::string Usuario::getEmail() const { return email; }
TipoUsuario Usuario::getTipo() const { return tipo; }